### ISSUE TITLE
syntax error, false positive with ALIASES

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1654]
`ajson->mt_json_tree` is allowed,

```abap
INTERFACE lif_ajson.
  DATA mt_json_tree TYPE string.
ENDINTERFACE.

CLASS lcl_ajson DEFINITION.
  PUBLIC SECTION.
    INTERFACES lif_ajson.
    ALIASES mt_json_tree FOR lif_ajson~mt_json_tree.
ENDCLASS.

CLASS lcl_ajson IMPLEMENTATION.
ENDCLASS.

FORM bar.
  DATA ajson TYPE REF TO lcl_ajson.
  WRITE ajson->mt_json_tree.
ENDFORM.
```

![image](https://user-images.githubusercontent.com/5888506/104089489-d434a700-526f-11eb-8ff7-ec9bccb1af98.png)
